File: domain/src/main/java/com/lowbottgames/au/perth/traffic/cam/domain/PlaceItem.java
package com.lowbottgames.au.perth.traffic.cam.domain;

public class PlaceItem {

    public String placeName;
    public CamItem[] camItem;

    public PlaceItem(String placeName, CamItem[] camItem) {
        this.placeName = placeName;

        this.camItem = camItem;
    }

    @Override
    public String toString() {
        return placeName;
    }
}

File: domain/src/main/java/com/lowbottgames/au/perth/traffic/cam/domain/PlaceItem.kt
package com.lowbottgames.au.perth.traffic.cam.domain

class PlaceItem {

    val placeName: String
    val camItem: Array<CamItem>

    constructor(
        placeName: String,
        camItem: Array<CamItem>
    ) {
        this.placeName = placeName
        this.camItem = camItem
    }

}
